lib/plugins/aws/lib/naming.js

```diff
@@ -545,8 +545,13 @@ module.exports = {
   getLambdaAtEdgeInvokePermissionLogicalId(functionName) {
     return `${this.getLambdaLogicalId(functionName)}InvokePermission`;
   },
-  getCloudFrontOriginId(functionName, path) {
-    let originId = this.getNormalizedFunctionName(functionName);
```

Comment:
Wouldn't it be breaking to change the originId, for existing stacks?

Maybe it'll be good to maintain same resolution for typical cases, and introduce new resolution for those which were not covered well (?)

Reply:
I've made a real tests from existing stack with old names to the new one and its only required CloudFront update - not breaking anything.

The goal of naming lib is ability to cross generate name for resources, if we add two different logic for generating names it wil be hard to achive that.